lambdas/preview/index.py

```diff
@@ -60,24 +59,24 @@ def lambda_handler(event, _):
 
     resp = requests.get(url)
     if resp.ok:
-        with NamedTemporaryFile() as fd:
```

Comment:
I kinda like `fd`, TBH. The trailing `_`s are weird. (If `pylint` is complaining about it, I can fix it.)

Reply:
Yeah, `pylint`. I find this linting rule a tad silly but, at the same time, `fd` is myster meat. `f` makes more sense to me but not religious.

Reply:
Well, `fd` just means "file descriptor". (I guess it's technically a file object here, not a file descriptor... need to find out what the convention is.)

Reply:
I think you can modify how permissive pylint is about different variable names if you want to allow things like fd: 
http://pylint.pycqa.org/en/1.9/user_guide/options.html#cmdoption-variable-rgx